lib/matplotlib/projections/polar.py

```diff
@@ -561,8 +560,7 @@ def set_rgrids(self, radii, labels=None, angle=None, fmt=None,
         if angle is None:
             angle = self.get_rlabel_position()
         self.set_rlabel_position(angle)
-        for t in self.yaxis.get_ticklabels():
-            t.update(kwargs)
+        self.yaxis.set_tick_params(which='major', **kwargs)
```

Comment:
Is this really equivalent (it probably should, but we all know how consistent the API is)?  set_tick_params ultimately forwards the arguments to Tick._apply_params, which is different from Tick.update (i.e. Artist.update).